utils/sparse.py

```diff
@@ -119,3 +166,17 @@ def qat_active(self, epoch):
             return qat_start < epoch + 1
 
         return False
+
+    def compose_recipes(self):
+        """
+        Combine checkpoint recipe + training recipe and return the new composed manager
+
+        :return: A manager representing composed recipe
+        """
+        if not self.checkpoint_recipe_manager:
+            return self.manager
+
+        return self.manager.compose_staged(
+            base_recipe=self.recipe, additional_recipe=self.checkpoint_recipe_manager,
```

Comment:
checkpoint recipe is the base recipe, the training recipe is additional (train is building off of the checkpoint base)

Reply:
Good catch, many thanks! addressed, tested and updated description accordingly.<img src="https://media0.giphy.com/media/xT9DPwQpQ9imTtYkla/giphy.gif"/>